content/en/blog/_posts/2023-09-15-Bootstrap-an-Air-Gapped-Cluster-with-Kubeadm/index.md

```diff
@@ -0,0 +1,665 @@
+---
+layout: blog
+title: "Bootstrap an Air Gapped Cluster With Kubeadm"
+date: 2023-09-15
+slug: bootstrap-an-air-gapped-cluster-with-kubeadm
+---
+
+**Author:** Rob Mengert (Defense Unicorns)
+
+# Introduction
```

Comment:
```suggestion
```

Don't use a first-level heading. Also, we avoid starting articles with any level of heading.